src/main/java/racingcar/Car.java

```diff
@@ -34,6 +45,13 @@ public int getPosition(int index) {
         return positions.get(index);
     }
 
+    public int getLastPosition() {
+        if(positions.isEmpty())
+            return -1;
```

Comment:
인텔리제이 자동정렬은 `윈도우: Ctrl + Alt + L`, `macOS: Cmd + Alt + L` 입니다 :) 
코드 전체적으로 포맷팅이 필요 할 것 같아요! 
그리고 if 문이 한줄이면 대괄호를 생략해도 되지만, 추후 수정할때 실수를 방지하기위해 대괄호를 쓰는건 어떠신가요? 

```suggestion
     if (positions.isEmpty()) {
            return -1;
        }
```